data/net/minecraft/world/entity/animal/allay/Allay.mapping

```diff
@@ -4,15 +4,15 @@ CLASS net/minecraft/world/entity/animal/allay/Allay
 	METHOD getSpinningProgress (F)F
 		ARG 1 partialTick
 	METHOD isDuplicationItem (Lnet/minecraft/world/item/ItemStack;)Z
-		ARG 1 itemStack
+		ARG 1 stack
 	METHOD removeInteractionItem (Lnet/minecraft/world/entity/player/Player;Lnet/minecraft/world/item/ItemStack;)V
 		ARG 1 player
-		ARG 2 itemStack
+		ARG 2 stack
 	METHOD setDancing (Z)V
 		ARG 1 dancing
 	METHOD setJukeboxPlaying (Lnet/minecraft/core/BlockPos;Z)V
 		ARG 1 jukeboxPos
-		ARG 2 starting
+		ARG 2 jukeboxPlaying
```

Comment:
Investigation and an explanation would be preferred.

Reply:
I don't see an issue with this. The method is named `setJukeboxPlaying` and this argument is what to set it to.